mne/preprocessing/xdawn.py

```diff
@@ -16,7 +16,7 @@
 from ..io.pick import _pick_data_channels
 from ..utils import logger
 from ..externals.six import iteritems, itervalues
-
+from ..stats.regression import _prepare_rerp_data, _prepare_rerp_preds, _solver
```

Comment:
It's faster for the scenarios I tested it on (which were, admittedly, a bunch). For the XDawn case, with a low number of sparse predictors and short time windows (compared to what I'm usually doing: many predictors, including a few dense ones, and some very long time windows), it should usually be really fast anyways.

If it's not sufficiently fast, I'll have to change. The modularity actually makes it really easy to swap in another solver.

The drawback I experienced with the pinv based one was that it's a bit less stable.